addon/mixins/osf-model.js

```diff
@@ -7,5 +7,37 @@ import DS from 'ember-data';
 
 export default Ember.Mixin.create({
     links: DS.attr('links'),
-    embeds: DS.attr('embed')
+    embeds: DS.attr('embed'),
+
+    relationshipLinks: DS.attr('links'),
+    dirtyRelationships: Ember.computed('_dirtyRelationships', function() {
+        var dirtyRelationships = this.get('_dirtyRelationships');
+        return Object.keys(dirtyRelationships).map((rel) => {
+            if (rel === 'files' || rel === 'node' || rel === 'user') {
```

Comment:
Right- but I'm not really comfortable baking the list of specific exceptions into a parent class. 

a) Parent classes aren't supposed to be model-specific, and b) I could imagine a future model with a relationship that we didn't want to exempt.

If we had a relationship mixin, it could define a simple field at the top level:
`relationshipsNotToTrackInFancyMagic = ['node']` 

...and that single field could be overridden/checked per subclass as appropriate. That would provide a simple hook to control this behavior in a model-specific way.

(to the implementer, it would look kind of like `Component.classNames`)